components/Playground/Schema.module.scss

```diff
@@ -0,0 +1,306 @@
+.array {
+  color: #ff8fc8;
+}
+.number {
+  color: #acb2ff
+}
+.string {
+  color: var(--green-03);
+}
+
+.integer {
+  color: #f8c272;
+}
+
+.schemaRole {
+  font-size: 14px;
+  line-height: 1.43;
+  color: var(--green-03);
+  border: none;
+  margin: 8px 8px 0 0;
+  padding: 6px 8px;
+  border-radius: 4px;
+  background-color: rgba(0,255,104,.16);
+}
+
+.schemaCode {
+  color: var(--white-01);
+  font-size: 14px;
```

Comment:
Next let's focus on creating a global font styling file like what we have in Deriv-app ([fonts.scss](https://github.com/binary-com/deriv-app/blob/master/packages/shared/src/styles/fonts.scss))
and access css font variables from there.